### PR TITLE
docs: add server definition in spec

### PR DESCRIPTION
**Pull request by luphieanza**
---
title: "add server definition in spec"
---
Let's [add a 'server' definition in our Spec under the Definitions section](https://github.com/asyncapi/spec/blob/master/spec/asyncapi.md#definitions)
---

**Related issue(s):**
Resolves website issue [#587](https://github.com/asyncapi/website/issues/587)

### CODE REVIEW

**Comment by luphieanza** [comment]
@alequetzalli @derberg Hi, Folks! Please have a check for this PR.

**Review by dalelane — DISMISSED** [review]
LGTM

File: spec/asyncapi.md [hunk 1]
```diff
@@ -84,20 +84,23 @@ It means that the [application](#definitionsApplication) allows [consumers](#def
 
 ## <a name="definitions"></a>Definitions
 
+#### <a name="definitionsServer"></a>Server
+A server MAY be a message broker that capable to sending and/or receiving between [producer](#definitionsProducer) and [consumer](#definitionsConsumer) but also a server MAY be a service with WebSocket API that enables message-drive communication between browser-to-server or server-to-server.
```
Comment (alequetzalli) [inline spec/asyncapi.md:88]:
```suggestion
A server may be a message broker that is capable of sending and/or receiving between a [producer](#definitionsProducer) and [consumer](#definitionsConsumer). A server may also be a service with WebSocket API that enables message-drive communication between browser-to-server or server-to-server.
```

caps for no reason are odd, so let's remove them ❤️😀👍🏽

**Review by alequetzalli — CHANGES_REQUESTED** [review]
@luphieanza this contribution is SO useful and helpful, thank you! You're a ⭐️ 

Just a tiny editorial comment and it looks like @dalelane approved the technical aspect.

Let's wait to see at least what @derberg or/and @fmvilas say. 👍🏽

File: spec/asyncapi.md (hunk 1 above)
Comment (luphieanza) [reply spec/asyncapi.md:88]: about 'MAY' word with capitalization I found it on here image
might be you want check this out , it should refer to interpreted as described in [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt), right?

Comment (alequetzalli) [reply spec/asyncapi.md:88]: hmmm... ok, let's have @derberg confirm the `MAY` question, but go ahead w/ the other changes 👍🏽

Comment (luphieanza) [reply spec/asyncapi.md]: Alright 👍🏽

File: spec/asyncapi.md [hunk 2]
```diff
@@ -84,20 +84,23 @@ It means that the [application](#definitionsApplication) allows [consumers](#def
 
 ## <a name="definitions"></a>Definitions
 
+#### <a name="definitionsServer"></a>Server
+A server MAY be a message broker that is capable of sending and/or receiving between a [producer](#definitionsProducer) and [consumer](#definitionsConsumer). A server MAY be a service with WebSocket API that enables message-drive communication between browser-to-server or server-to-server.
```
Comment (derberg) [inline spec/asyncapi.md:88]:
```suggestion
A server MAY be a message broker that is capable of sending and/or receiving between a [producer](#definitionsProducer) and [consumer](#definitionsConsumer). A server MAY be a service with WebSocket API that enables message-driven communication between browser-to-server or server-to-server.
```

Comment (luphieanza) [reply spec/asyncapi.md]: @derberg How about the "MAY" caps word is it true that I need to make it caps? cc. @alequetzalli

Comment (derberg) [reply spec/asyncapi.md]: yeap, it should be caps

**Review by derberg — APPROVED** [review]

**Review by alequetzalli — APPROVED** [review]
LGTM 🎉🎉🎉🎉

**Comment by derberg** [comment]
@all-contributors please add @luphieanza for docs

**Pull request merged**